project-api-tests/src/test/java/org/ballerina/projectapi/HierarchicalPackagesTest.java

```diff
@@ -147,10 +148,17 @@ private void pushUpdatedVersion(List<String> previousVersions, int i, String pac
         }
     }
 
+    private void replaceRandomPackageName(String randomPackageName, String packageName) throws IOException{
+        List<Path> balFiles = CentralTestUtils.getFileListByExtension(tempWorkspaceDirectory, "bal");
+        List<Path> tomlFiles = CentralTestUtils.getFileListByExtension(tempWorkspaceDirectory, "toml");
```

Comment:
I think It is good to introduce constants for these extensions. (`bal` and `toml`)